core/src/main/scala/org/apache/spark/util/ClosureCleaner.scala

```diff
@@ -27,6 +26,7 @@ import org.apache.xbean.asm7.Opcodes._
 
 import org.apache.spark.{SparkEnv, SparkException}
 import org.apache.spark.internal.Logging
+import org.apache.spark.util.Utils.getSerializedLambda
```

Comment:
since we are already in `org.apacke.spark.util`, could we just call `Utils.getSerializedLambda` from the `clean`? We seem to refer to other static functions in `Utils` using the classname.